RockPaperScissors/Entities/Game.cs

```diff
@@ -40,6 +40,35 @@ public GameResult Play()
 
         private bool Attack(Player from, Player to)
         {
+            bool sonarTest = false;
+            if (sonarTest)
+                return false;
+
+            bool ssss = false;
+            if (ssss)
+                return false;
+            else if (ssss == true)
+                return true;
+
+            bool sonarTest3 = false;
+            if (sonarTest3)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to 'false'; some subsequent code is never executed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=csharpsquid%3AS2583)